src/uipath/_cli/_evals/_helpers.py

```diff
@@ -0,0 +1,194 @@
+# type: ignore
+import ast
+import importlib.util
+import json
+import logging
+import re
+import sys
+from pathlib import Path
+from typing import Any, Optional
+
+import click
+
+from uipath._cli._utils._console import ConsoleLogger
+from uipath._utils.constants import CUSTOM_EVALUATOR_PREFIX
+
+logger = logging.getLogger(__name__)
+console = ConsoleLogger().get_instance()
+
+
+def try_extract_file_and_class_name(text: str) -> tuple[bool, str, str]:
+    if text.startswith(CUSTOM_EVALUATOR_PREFIX):
+        file_and_class = text[len(CUSTOM_EVALUATOR_PREFIX) :]
+        if ":" not in file_and_class:
+            raise ValueError(
+                f"evaluatorSchema must include class name after ':' - got: {text}"
+            )
+        file_path_str, class_name = file_and_class.rsplit(":", 1)
+
+        return True, file_path_str, class_name
+    return False, "", ""
+
+
+def to_kebab_case(text: str) -> str:
+    return re.sub(r"(?<!^)(?=[A-Z])", "-", text).lower()
+
+
+def find_evaluator_file(filename: str) -> Optional[Path]:
+    """Find the evaluator file in evals/evaluators/custom folder."""
+    custom_evaluators_path = Path.cwd() / "evals" / "evaluators" / "custom"
+
+    if not custom_evaluators_path.exists():
+        return None
+
+    file_path = custom_evaluators_path / filename
+    if file_path.exists():
+        return file_path
+
+    return None
+
+
+def find_base_evaluator_class(file_path: Path) -> Optional[str]:
```

Comment:
This basically fails if there are more than one evaluator. Should we support something like `foo.py:Bar` where `Bar` is the classname?